### ISSUE TITLE
When TI Terrain Map can be tradable or gift?

### ISSUE THREAD

**Issue by AgentJumpz** [ValveSoftware/Dota2-Gameplay#6766]
### Description

**I think if the terrain is not obtainable again it should be tradable or gifted or traded once. 
Most people want to find it but it's not tradable or some people don't use it.
It's useless to permanently add an account. New players need it.
Is it possible to bring it back for sale? Besides, it only sees the single player side, please make it sellable.
Don't let us use mods again, if you don't want to bring them back for sale please make them tradable.**

Thank you.

**Comment by shaileshgarg** [comment]
Not tradeable, I would like to get Immortal Gardens map though.

**Comment by AgentJumpz** [comment]
Yes me too. I would like to get all terrain map. please let it to tradable or marketable

**Comment by andraip** [comment]
This is not a technical issue. Post this on Reddit or something instead of wasting the time of the devs who want to fix bugs in Dota.
